packages/core/src/components/date-picker/date-picker.tsx

```diff
@@ -26,55 +26,75 @@ import { DateTimeCardCorners } from '../date-time-card/date-time-card';
   scoped: true,
 })
 export class DatePicker {
+  private daysInWeek = 7;
+  private dayNames = Info.weekdays();
+  private monthNames = Info.months();
+
   /**
-   * output date format
+   * Date format string.
+   * See {@link https://moment.github.io/luxon/#/formatting?id=table-of-tokens} for all available tokens.
    */
   @Prop() format: string = 'yyyy/LL/dd';
 
   /**
-   * Set range size
+   * If true a range of dates can be selected.
    */
   @Prop() range: boolean = true;
 
   /**
-   * set styles
+   * @deprecated - will get removed with next major release
    */
   @Prop() individual: boolean = true;
 
   /**
-   * Set corners style
+   * Corner style
    */
   @Prop() corners: DateTimeCardCorners = 'rounded';
 
-  private daysInWeek = 7;
-  private dayNames = Info.weekdays();
-  private monthNames = Info.months();
-  @State() year = DateTime.now().year;
-  @State() month = DateTime.now().month;
-  @State() calendar: [number, number[]][] = [];
+  /**
+   * Year to display initially.
+   */
+  @Prop() year = DateTime.now().year;
+
+  /**
+   * Month to display initially.
+   */
+  @Prop() month = DateTime.now().month;
+
```

Comment:
Day is missing or did i missed something?
What iam also not understand how can you select a date range?

We should also add minDate maxDate minTime maxTime (https://github.com/siemens/ix/issues/50) because it was a feature from the original datepicker which are frequently used